app/src/main/java/com/example/unsplashphotos/data/RetrofitInstance.kt

```diff
@@ -0,0 +1,32 @@
+package com.example.unsplashphotos.data
+
+import com.google.gson.GsonBuilder
+import okhttp3.OkHttpClient
+import okhttp3.logging.HttpLoggingInterceptor
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+class RetrofitInstance {
```

Comment:
no need of this file. expose okhttp and retrofit instances using provides in Hilt